src/main/java/com/feedzai/commons/sql/abstraction/util/Constants.java

```diff
@@ -84,4 +84,12 @@
      * server. This is only possible on implementations that support this behavior. The default value is {@code false}.
      */
     public static final boolean DEFAULT_DISABLE_LOB_CACHING = false;
+    /**
+     * The default login socket timeout.
```

Comment:
login socket?

Reply:
It's the timeout of making a connection to the socket. Because, the database can fail to connect. I will describe it better in javadocs.

Reply:
Suggestion: Default duration (in seconds) to wait for the database connection to be established.

Reply:
Thanks for the suggestion. You are right. That way makes more sense.